README.md

```diff
@@ -85,15 +95,14 @@ library. The interface for that library is described in
 which implements the `nabto_device` library and apps/examples which
 are using the `nabto_device` library.
 
-For help on porting of the nabto_device.h interface to a new platform
-see `doc/platform_integration_howto.md` and
-`platform_integration_steps/README.md`
+For help on porting of the `nabto_device.h` interface to a new platform, see the Nabto Edge Embedded
```

Comment:
I see this is just a reformatting of the same words, but would it not be "porting of the `nabto_device` library" not the interface?